pennylane/templates/subroutines/qft.py

```diff
@@ -93,7 +93,7 @@ def _matrix(cls, num_wires):
         return mat / np.sqrt(dimension)
 
     @staticmethod
-    def decomposition(wires):
+    def compute_decomposition(wires):
```

Comment:
Not urgent, but is there a reason this one doesn't get a docstring?

Reply:
Did docstrings for qubit `ops` and forgot I updated a template. Thanks for the reminder.